app/server/tests/api-health.test.js

```diff
@@ -0,0 +1,18 @@
+/*  
+    Testing to ensure API is running and correct response is given.
+    Endpoint is /api/health
+*/
+
+const endpoint = 'https://hwp-express-api-d63404-dev.apps.silver.devops.gov.bc.ca/api/health';
```

Comment:
we should be getting this value from an environment variable if possible please. Hard coding values like this could lead to security vulnerabilities.

Reply:
Done. Configuration for .env with Jest is odd. Had to edit the package.json, and I don't see a way to specify location, so it looks for .env at the server root.